tests/functional/c/consider/consider_using_with_open.py

```diff
@@ -137,7 +137,7 @@ def test_defined_in_try_and_finally(self):
         except FileNotFoundError:
             Path("foo").touch()
         finally:
-            file_handle.open("foo", encoding="utf")  # must not trigger
+            file_handle.open("foo", encoding="utf")  # [used-before-assignment] # must not trigger
```

Comment:
Is it a new false positive or should the comment be removed ?

Reply:
I think `used-before-assignment` is correct, but `consider-using-with` is apparently not supposed to be triggered. I can update the comment!

Reply:
Ha, I get it
```suggestion
            # +1: [used-before-assignment] 
            file_handle.open("foo", encoding="utf")  # must not trigger consider-using-with
```